src/main/java/com/academy/ui/components/AddClubPopUpComponent/AddClubPopUpStepOne.java

```diff
@@ -70,12 +74,18 @@ public class AddClubPopUpStepOne extends AddClubPopUpContainer {
     @FindBy(xpath = "./descendant::span[@class='ant-select-selection-item']")
     private WebElement centerSelectedTitle;
 
-    @FindBy(xpath = "//div[@class='ant-select-item-option-content']")
+    @FindBy(xpath = "//div[@class='ant-select-item ant-select-item-option']")
```

Comment:
This xpath miss selected item of the list. It is better use 'contains'
//div[contains(@class,'ant-select-item ant-select-item-option')]